src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -96,6 +98,20 @@ public RoutingResponse route(Router router) {
             routingErrors.addAll(e.getRoutingErrors());
         }
 
+        // Direct flex routing
+        if (OTPFeature.FlexRouting.isOn()) {
+            try {
+                itineraries.addAll(DirectFlexRouter.route(
+                    request,
+                    ADDITIONAL_SEARCH_DAYS_BEFORE_TODAY,
+                    ADDITIONAL_SEARCH_DAYS_AFTER_TODAY
```

Comment:
These 2 parameters are passed inn with the request in several places, if we move them to the request they no longer need to be passed in. This would also make these parameters configurable.